src/Sulu/Bundle/AdminBundle/ResourceMetadata/ResourceMetadataPool.php

```diff
@@ -16,10 +16,20 @@
 class ResourceMetadataPool
 {
     /**
-     * @var FormResourceMetadataProvider[]
+     * @var ResourceMetadataProviderInterface[]
      */
     private $resourceMetadataProviders;
 
+    /**
+     * Returns all the registered providers.
+     *
+     * @return ResourceMetadataProviderInterface[]
+     */
+    public function getProviders(): array
```

Comment:
The only usage of this method I can find is in the tests... Do we really need it?